packages/websocket-lwt-unix/websocket-lwt-unix.2.13/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+name: "websocket-lwt-unix"
+version: "2.13"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/ocaml-websocket"
+bug-reports: "https://github.com/vbmithr/ocaml-websocket/issues"
+dev-repo: "git+https://github.com/vbmithr/ocaml-websocket"
+doc: "https://vbmithr.github.io/ocaml-websocket/doc"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
```

Comment:
This is not a blocker at all but just for the information `"@install"` is not necessary